variables.tf

```diff
@@ -126,8 +126,8 @@ variable "bucket_versioning_enabled" {
 
 variable "bucket_force_destroy" {
   type        = bool
-  default     = false
-  description = "Force destroy bucket (Required when bucket not empty)"
+  default     = true
+  description = "Force destroy bucket (if disabled, terraform will not be able do destroy non-empty bucket)"
```

Comment:
If we apply the above change to description we need to apply here too.

Reply:
I agree, @dmurray-lacework Would you mind updating the rest of the modules to match this description. Please?

Reply:
```suggestion
  description = "Force destroy bucket (When 'false' a non-empty bucket will NOT be destroyed.)"
```

Reply:
Created a task, we can address this in future since most of the PR's have been merged.
https://lacework.atlassian.net/browse/GROW-2293